stdlib/public/core/DebuggerSupport.swift

```diff
@@ -328,6 +328,26 @@ public enum _DebuggerSupport {
 
     return target
   }
+
+  public static func stringForPrintObject(_ pointer: UnsafeRawPointer?, mangledTypeName: String) -> String {
+    guard let pointer else { return "invalid value pointer" }
+
+    guard let type =
+      unsafe _getTypeByMangledNameInContext(
+        mangledTypeName,
+        UInt(mangledTypeName.count),
+        genericContext: nil,
```

Comment:
Since we're passing `nil` here, does it mean this doesn't work with generic types the moment?